include/electionguard/voting/coordinator.h

```diff
@@ -16,11 +16,14 @@ enum Voting_Coordinator_status
     VOTING_COORDINATOR_SUCCESS,
     VOTING_COORDINATOR_INSUFFICIENT_MEMORY,
     VOTING_COORDINATOR_INVALID_BALLOT_ID,
+    VOTING_COORDINATOR_INVALID_BALLOT_INDEX,
     VOTING_COORDINATOR_INVALID_BALLOT,
     VOTING_COORDINATOR_UNREGISTERED_BALLOT,
     VOTING_COORDINATOR_DUPLICATE_BALLOT,
     VOTING_COORDINATOR_TIMED_OUT_BALLOT,
+    VOTING_COORDINATOR_END_OF_FILE,
     VOTING_COORDINATOR_IO_ERROR,
+    VOTING_COORDINATOR_ERROR_ALREADY_EXISTS,
```

Comment:
It is recommended to add new entries of the enum at the end to avoid changing the value for existing status,  or if you want to keep related status IDs together, establish a range.

Reply:
good catch.  thanks!